source/admin_manual/mailbox_formats/mbox.rst

```diff
@@ -8,14 +8,16 @@ For information on how to configure mbox in Dovecot, see :ref:`mbox_settings`.
 
 .. warning::
 
-  Mbox format is considered deprecated and is maintained primarily for
-  backwards compatibility and utility purposes (specifically for archival
-  purposes, as mbox allows multiple messages to be natively stored in a
-  single file).
+  Mbox format is considered deprecated and nobody should really be using it
+  anymore for daily use.
```

Comment:
Make even stronger: "Mbox format is deprecated and should not be used in production setups."